src/Core/Context/CurrentContext.cs

```diff
@@ -84,7 +84,12 @@ public async virtual Task BuildAsync(HttpContext httpContext, GlobalSettings glo
 
         if (httpContext.Request.Headers.ContainsKey("Bitwarden-Client-Version"))
         {
-            ClientVersion = new Version(httpContext.Request.Headers["Bitwarden-Client-Version"]);
+            // The string.Split method is being used for cases where QA builds include the git hash e.g. '2022.10.2 - 5f8c1c1'
+            var clientVersionHeader = httpContext.Request.Headers["Bitwarden-Client-Version"].ToString().Split(' ')[0];
```

Comment:
Do we ened to null check before doing `ToString()`

Reply:
I removed this and created a PR for the client version to not send the git hash. https://github.com/bitwarden/clients/pull/4008